app/apis/api_kubernetes.py

```diff
@@ -747,6 +763,145 @@ def create_or_update_drucker_on_kubernetes(
             body=v2_beta1_horizontal_pod_autoscaler
         )
     """
+    return service_name
+
+def switch_drucker_service_model_assignment(
```

Comment:
It would be great if unit test for this method could also be added

Reply:
Yes I think so.
But this PR is not for modifying the unit test. We will create another one for this.